docs/administration.platforms.md

```diff
@@ -1,37 +1,70 @@
 ---
 id: administration.platforms
-title: Platform Operation
+title: meshPlatforms
 ---
 
-Operators can perform various maintainance operations for their platforms. For maintenance of Landing Zones, please see
-[Landing Zone Administration](meshcloud.landing-zones.md).
+A key task for Platform Operators is integrating your Cloud Platforms as meshPlatforms into meshStack. This enables
```

Comment:
"as meshPlatforms" -> I would just add a short explanation on what that means. I could imagine first time users being a bit confused by the terminology.

Reply:
added a sentence before